onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -199,13 +163,38 @@ def calculate_quantization_params(model, nbits=8, quantization_thresholds=None):
 
     return quantization_params
 
+
+def load_pb_file(data_file_name, dataset_size, height, width):
+    '''
+    Load tensor data from pb files.
+    :param data_file_name: path to the pb file
+    :param dataset_size: number of image-data in the pb file for data size check
+    :param height: image height for data size check
+    :param width: image width for data size check
+    :return input data for the model
+    '''
+    tensor = onnx.TensorProto()
+    inputs = np.empty(0)
+    with open(data_file_name, 'rb') as fin:
+        tensor.ParseFromString(fin.read())
+        inputs = numpy_helper.to_array(tensor)
+        try:
+            shape = inputs.shape
+            inputs = inputs.reshape(dataset_size, 1,3,height, width)
+        except:
+            sys.exit("Input .pb file contains incorrect input size. \nThe required size is: (%s). The real size is: (%s)"
+                        %((dataset_size, 1,3,height, width), shape))
+
+    return inputs
+
 def main():
     # Parsing command-line arguments
     parser = argparse.ArgumentParser(description='parsing model and test data set paths')
     parser.add_argument('--model_path', required=True)
     parser.add_argument('--dataset_path', required=True)
     parser.add_argument('--calib_mode', default='naive')
     parser.add_argument('--dataset_size', type=int, default=30)
+    parser.add_argument('--data_preprocess', type=str, default='')
```

Comment:
add usage around this argument so that users can understand how to select the pre-processing method.

Also update the readme doc

Reply:
Will do it.